pkg/registers/marshalling_test.go

```diff
@@ -1,10 +1,11 @@
 package registers_test
 
 import (
-	"github.com/9elements/converged-security-suite/v2/pkg/hwapi"
-	"github.com/9elements/converged-security-suite/v2/pkg/registers"
 	"reflect"
 	"testing"
+
+	hwInternal "github.com/9elements/converged-security-suite/v2/pkg/hwapi"
```

Comment:
I guess hwInternal is not in go-style and hwapi looks a bit better (I might be wrong)

Reply:
addressed